src/v2i-hub/CommandPlugin/src/CommandPlugin.cpp

```diff
@@ -633,7 +633,7 @@ int CommandPlugin::WSCallbackBASE64(
 											argsList[arg.first] = arg.second.data();
 										}
 									}
-									catch (exception argsEx)
+									catch (exception &argsEx)
```

Comment:
```cpp
catch(const std::exception &argEx)
```

Also where you just address a code smell or why are there changes to this file at all.

Reply:
This was just to address a code smell